kyaml/setters2/set.go

```diff
@@ -200,15 +205,23 @@ func validateAgainstSchema(ext *cliExtension, sch *spec.Schema) error {
 	sc.Properties[ext.Setter.Name] = *sch
 
 	input := map[string]interface{}{}
-	format := `{"%s" : "%s"}`
-	if yaml.IsValueNonString(ext.Setter.Value) {
-		format = `{"%s" : %s}`
+	inputJSON := ""
+	format := ""
```

Comment:
Does this need to be defined here? Seems like it could be defined inside each if block.

Reply:
Right. Done.